packages/reg-notify-chatwork-plugin/src/chatwork-notifier-plugin.ts

```diff
@@ -0,0 +1,90 @@
+import { PluginCreateOptions, PluginLogger, NotifierPlugin, NotifyParams } from "reg-suit-interface";
+
+import { sendWebHook } from "./send-web-hook";
+
+export interface ChatworkNotiferPluginOptions {
+  roomID: string;
+  chatworkToken: string;
+  mention?: string;
+  message?: string;
+  mrURL?: string;
+  pipelineURL?: string;
```

Comment:
[nits] reg-suit's other plugins use not `URL` but `Url` suffix convention (e.g. `webhookUrl` or `GitHubUrl` ).
Would you rename to `mrUrl` / `pipelineUrl` ?